cmd/log-splicer/main.go

```diff
@@ -135,6 +135,19 @@ func copySecret(identityFolder string, b []byte) error {
 	return err
 }
 
+func copyFile(src, outdir string) error {
```

Comment:
maybe put a note that this is only okay for small files or rather put a `Stat` before `ReadFile` and abort when it's very big.


for the latter it's better to use `srcf = os.Open`, `dstf = os.Create(dst)` and use `io.Copy(dstf, srcf)` to not buffer the whole content to ram.

Reply:
done! totally bungled up io.Copy on the first attempt though and wondered why it wasn't working. The order of `(dst, src)` fucked me up lol